File: fizzbuzz.java
public class fizzbuzz {
    public static void main(String[] args) {
        for (int i = 1; i < 101; i++) {
            if ((i % 3 == 0) && (i % 5 == 0)) {
                System.out.print("fizzbuzz ");
            }
            else if (i % 3 == 0) {
                System.out.print("fizz ");
            }
            else if (i % 5 == 0) {
                System.out.print("buzz ");
            }
            else {
                System.out.print(i + " ");
            }
        }
    }
}

File: fizzbuzz.scala
object FizzBuzz {
  def main(args: Array[String]) {
    for (i <- 1 to 100) {
      if ((i % 3 == 0) && (i % 5 == 0)) print("fizzbuzz ")
      else if (i % 3 == 0) print("fizz ")
      else if (i % 5 == 0) print("buzz ")
      else print(i + " ")
    }
  }
}